samples/quickstart.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright 2020, Google LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+async function main() {
+  // [START bigquery_storage_quickstart]
+  async function bigqueryStorageQuickstart() {
+    // The read stream contains blocks of Avro-encoded bytes. We use the
```

Comment:
Lines 21 through 31 should go *outside* of the async function, and inside of `main`.